### DISCUSSION TITLE
Roundabout?

### DISCUSSION THREAD

**Discussion by folknor** [Opinionated-Blueprints/10-Books-Full-of-Rails#63] [Ideas]
Hi

Would it be possible to make a roundabout? I want an intersection where trains can turn around and head in the opposite direction :-)

Thank you!

**Comment by brandon-gradient** [comment]
Pretty sure you can use the "u-turn" rotated and pasted in all 4 directions to create what you're talking about. I know there are some straights between the 90 degree turns but I think it would work out. If I'm wrong it seems like modifying that blueprint so that it does work like this would be a nice touch

**Comment by Bocian-1** [comment]
Roundabouts are a no-go. It's asking for a deadlock and vastly inferior throughput. Trains also almost never need to do a U-turn unless the station exit is designed in a way where the path literally doesn't exist. For those cases, there are U-Turns. Roundabouts are also bad for UPS.
